hoot-services/Makefile

```diff
@@ -62,6 +62,7 @@ clean-db-force:
 	rm -rf ./src/main/java/hoot/services/db2
 
 test: config
+	echo Running services tests...
```

Comment:
I can remove it....not a big deal.  My personal dislike now, however, is that so much has been stripped from the test output that when you run all of the tests in silent mode, its hard to tell what tests are running and when.  When running the services tests now, for example, all you see in the final output is "TESTS"...not what tests actually ran.  If you remove silent, then you get the firehose and is very hard to sift through all that.  I guess the workaround could be to output test text to file when you really need to get more information about the sequence they ran in.